db/schema.rb

```diff
@@ -19,6 +19,7 @@
     t.string "name", null: false
     t.text "body", null: false
     t.float "rating", default: 0.0, null: false
+    t.string "photo_path"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
   end
```

Comment:
Also I just noticed this now, but ideally anything relating to photo uploads should have been in this feature. So you could have added both users photos and animal photos. I totally missed it earlier, my bad! (line 33)

Reply:
I'm going to add this note here because I can't remember, but you will have to make sure you devise user tests are updated to include this photo uploader